inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -227,14 +227,16 @@ def set_wf_not_completed_ids_to_wf(obj, skip_blocked=True, skip_halted=False):
         skip_halted: boolean, if True, then it skips HALTED workflows when
             looking for matched workflows
     """
+    def _ignore_core_selection(base_record, match_result):
+        return get_value(match_result, '_source._workflow.workflow_name') != "CORE_SELECTION"
```

Comment:
I would restrict this to `Article` actually. It's probably a bug that we're matching `edit_article` here.

Reply:
:+1 I have to fix few tests now as this change is breaking them ;)